src/Host/Broker/Impl/Sessions/Session.cs

```diff
@@ -83,11 +83,12 @@ public class Session {
 
             var useridentity = User as WindowsIdentity;
             // In remote broker User Identity type is always WindowsIdentity
-            string suppressUI = (useridentity == null) ? string.Empty : "--suppress-ui ";
+            string suppressUI = (useridentity == null) ? string.Empty : "--rhost-suppress-ui ";
+            string isRepl = Id.StartsWithIgnoreCase("REPL") ? string.Empty : "--rhost-interactive ";
```

Comment:
As long as this is not too expensive. We want this change in for the BB tomorrow.